### ISSUE TITLE
Allow alt text on images to be indicative of decorative images

### ISSUE THREAD

**Issue by danahertzberg** [az-digital/az_quickstart#3657]
## Motivation
I want to add imagery to my website and be able to identify it appropriately with alt text.

### Is your feature request related to a problem? Please describe.
Currently, the alt text field is required but there is not a way to indicate the image as decorative.
Identified in this PR: https://github.com/az-digital/az_quickstart/pull/3610#issuecomment-2293887879

Alt text should be added to every image, but if there is a space in the alt text content, a screen reader will skip over it.
If alt text does not exist at all, the screen reader will read out the title of the image which is not ideal.

### Describe the solution you'd like
Final solution to be determined.

### Describe alternatives you've considered
There are a number of options we could use to help this issue.

1. The alt text field could allow ` ` as a valid text input
2. There could be a checkbox to mark the image as "decorative" which puts the space in the alt text field.
3. There could be a ` ` in the alt text field by default.

Regardless of solution, help text on the alt text field should be added so more context is provided to the content editor.

**Comment by ewlyman** [comment]
When the Decorative Image Widget module is installed and configured it presents users with two actions, one of which must always be completed before saving image content:

- Adding alternative text 
OR

- Selecting "Decorative"

There are, however, some quirks --for example, selecting "Decorative" for an image that already has alternative text does not immediate clear the alternative text, but will do so the _next_ time the image is saved.

BradBarkel's comment on Alt text being context dependent is also a concern. Site builders can get around this by uploading a new copy of an image, but having many duplicate images in the Media Library may pose problems.

I will set up a draft PR to demonstrate how Decorative Image Widget integrates with QuickStart, so we have a basis for further discussion. I think we are still not ready to call this contrib module our solution, however.

**Comment by trackleft** [comment]
Thanks for this, I saw this issue here in one of your references that might be interesting. https://www.drupal.org/project/decorative_image_widget/issues/3384144

**Comment by ewlyman** [comment]
@trackleft  --Yes, this IS interesting. Very.
I explored it a little towards the end of last week and it's promising. There is a catch, however:

Our Quickstart image media type uses the field  field_media_az_image. field_meia_az_image. https://www.drupal.org/project/media_library_media_modify creates its own image field, which is associated in the database to per-content-item information. We'd need to pull out the code and configuration supporting that functionality and field storage and incorporate it into our az_media module (I think).

Definitely worth exploring, but also potentially a bigger lift. Maybe for the new feature request?

Please let me know what you think.
